darwin/dataset/remote_dataset.py

```diff
@@ -374,15 +374,42 @@ def create_annotation_class(self, name: str, type: str):
                 "name": name,
                 "metadata": {"_color": "auto"},
                 "annotation_type_ids": [type_id],
+                "datasets": [{"id": self.dataset_id}],
```

Comment:
Do we still need `"dataset_id"` at line 373?

Reply:
strangely enough both are needed. @begedin maybe we could change that?